config/software/flapjack.rb

```diff
@@ -22,45 +25,39 @@
 
```

Comment:
and the gem_home fixes, and putting double quotes around the env values.

Other than that, yeah, line endings and indentation.